das/daser.go

```diff
@@ -17,25 +18,39 @@ import (
 
 var log = logging.Logger("das")
 
-// DASer continuously validates availability of data committed to headers.
-type DASer struct {
-	ctx    context.Context
-	cancel context.CancelFunc
+const (
+	priorityBufferSize = 1024
+	concurrency        = 256
+	jobsBufferSize     = 16
+	storeInterval      = 10 * time.Minute
+)
```

Comment:
Should we make it private? It can't be used outside really and pollutes Public API

Reply:
What are you pointing to? The `Config`?

Reply:
Yes, I guess this is the issue with WEB VScode I've used for reviewes. Its buggy unfortunately